src/SignaturePad.Forms.Platform.Shared/SignaturePadCanvasRenderer.cs

```diff
@@ -162,7 +166,11 @@ private void OnPointsSpecified (object sender, SignaturePadCanvasView.PointsEven
 			var ctrl = Control;
 			if (ctrl != null)
 			{
+#if NET471
+				ctrl.LoadPoints (e.Points.Select (p => new NativePoint ((int)p.X, (int)p.Y)).ToArray ());
```

Comment:
If it is possible to use `PointF` and `SizeF`, then nothing has to change here.

Reply:
stylus point works ok with float so it is redundant and removed